src/PKSim.Core/Services/SimulationAnalysisCreator.cs

```diff
@@ -52,14 +55,52 @@ public ISimulationAnalysis CreateAnalysisFor(Simulation simulation)
          }
       }
 
+      public ISimulationAnalysis CreatePredictedVsObservedAnalysisFor(IndividualSimulation simulation)
+      {
+         try
+         {
+            if (simulation != null)
+            {
+               _simulationAnalysis = _chartFactory.CreatePredictedVsObservedChartFor(simulation);
```

Comment:
WHY are you setting the gloval variable if you are not using any of the private methods? This codes looks way too complicated for hat it does

```
var simulationAnalysis = _chartFactory.CreatePredictedVsObservedChartFor(simulation);
AddSimulationAnalysisTo(simulation, simulationAnalysis);
return simulationAnalysis
```
no?

Reply:
true